CONTRIBUTING.md

```diff
@@ -0,0 +1,131 @@
+By participating in the development of GatorGrouper, you will be expected to
+adhere to this guide to the best of your ability.
+
+## Contributing bug fixes
+
+If you have found a bug:
+ * First, check the [Issue Tracker](https://github.com/GatorEducator/gatorgrouper/issues)
+   to see if the bug has already been found!
+ * Once you are sure there is no existing Issue, you can
+   [open a new issue](https://github.com/GatorEducator/gatorgrouper/issues/new).
+   Write a **clear, descriptive title**, any information relevant to the bug
+   (replicating the bug, specific usage problems, etc.), and provide a **code
+   sample** or **executable test case** that demonstrates the anomalous
+   behavior.
+
+If you have written a patch for a bug:
+  * Check the Issue Tracker for any relevant issues related to the bug.
+  * If there are no relevant issues, [raise one](https://github.com/GatorEducator/gatorgrouper/issues/new).
+    Indicate that you have written a patch for this bug.
+  * Once you have found an issue or raised one, you can then [open a new pull
+    request](https://github.com/GatorEducator/gatorgrouper/compare). Write a
+    **clear description** of the bug and how you have patched it. Reference any
+    relevant issues, or the new issue you have raised for the fix.
+  * Check the [coding standards](#coding-standards) section to ensure that you
+    have followed the coding conventions of GatorGrouper.
+
+## Contributing features
+
+If you have implemented a new feature for GatorGrouper:
+  * Check the Issue Tracker for any issues relevant to your implemented feature.
+  * [Open a new pull request](https://github.com/GatorEducator/gatorgrouper/compare).
+    Write a **clear description** of the feature and its usage. In your pull
+    request, be sure to have updated any relevant documentation to reflect the
+    usage of the tool with your implemented features.
+  * Be sure that you have followed the [coding standards](#coding-standards)
+    outlined below for any code contributions you have made.
+
+## Coding standards
+
+By contributing source code to the GatorGrouper project, you are agreeing that
+any code you have written follows the coding standards outlined below. This
+means adhering to the style, testing, and complexity standards as follows.
+
+### Code style
+
+All GatorGrouper code should be written in Python. Code written for GatorGrouper
+should,
+  * Be written in a modular fashion, making use of functions and modules.
+  * Be clearly documented with both docstring comments describing functions, as
+    well as single line comments describing particularly complex lines of code.
+  * Pass `flake8` checks without errors. You can learn more about using `flake8`
+    [here](http://flake8.pycqa.org/en/latest/).
+  * Score 10.0/10.0 when using the `pylint` tool. You can learn more about using
+    `pylint` [here](https://www.pylint.org/).
+
+It is strongly encouraged that you use the Python code formatter,
```

Comment:
Is it not required? `.travis.yml` fails anything that is not in the correct `black` format.

Reply:
I was unsure of the relation between `black` and `pylint`, and had assumed that the only way to get a 10.0/10.0 on `pylint` was to use `black`. Is this incorrect? If so, I will make the use of the code formatter `black` more overtly necessary.

Reply:
It is not strictly correct -- they are related, but do not require one another. I would mention both.

Reply:
Okay, I have disambiguated the black-pylint requirements in a new commit. @Michionlion review at your disclosure.